docs/my-website/docs/providers/mistral.md

```diff
@@ -44,14 +44,13 @@ for chunk in response:
 ## Supported Models
 All models listed here https://docs.mistral.ai/platform/endpoints are supported. We actively maintain the list of models, pricing, token window, etc. [here](https://github.com/BerriAI/litellm/blob/c1b25538277206b9f00de5254d80d6a83bb19a29/model_prices_and_context_window.json).
 
-| Model Name               | Function Call                                                                                                                                                      |
-|--------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| mistral-tiny | `completion(model="mistral/mistral-tiny", messages)` | 
-| mistral-small | `completion(model="mistral/mistral-small", messages)` | 
-| mistral-medium | `completion(model="mistral/mistral-medium", messages)` | 
-| mistral-large-latest | `completion(model="mistral/mistral-large-latest", messages)` | 
-| open-mixtral-8x22b | `completion(model="mistral/open-mixtral-8x22b", messages)` | 
-
+| Model Name     | Function Call                                                |
+|----------------|--------------------------------------------------------------|
+| Mistral Small  | `completion(model="mistral/mistral-small-latest", messages)` |
+| Mistral Large  | `completion(model="mistral/mistral-large-latest", messages)` |
```

Comment:
missing `mistral-medium-latest`

Reply:
I did not add it as Mistral Medium will be deprecated in the coming months:

<img width="828" alt="image" src="https://github.com/BerriAI/litellm/assets/4735661/0be3f322-8254-46f0-af0e-c82663a6f7f1">

Let me know, i can add that in no time.

Reply:
please add it - we support it today

Reply:
Added, thanks for guidance :)